widgets/src/components/ProcessDefinition/ProcessDefinitionContainer.js

```diff
@@ -48,9 +48,19 @@ class ProcessDefinitionContainer extends React.Component {
         throw widgetConfigs.errors[0];
       }
       const { config } = widgetConfigs.payload;
+      const settings = (config.settings && JSON.parse(config.settings)) || {};
+
+      const parsedSettings = Object.keys(settings).reduce(
+        (acc, settingKey) => ({
+          ...acc,
+          [settingKey]: JSON.parse(settings[settingKey]),
```

Comment:
I'm not sure if it's necessary to `JSON.parse()` child properties here as the entire object has already been parsed. Am I missing something here?